uber/templates/preregistration/form.html

```diff
@@ -48,8 +56,10 @@
             $('input[name="badge_type"]').val('{{ c.ATTENDEE_BADGE }}');
             $('.badge_type_selector').removeClass('active');
             $('.badge_type_selector').slice(badgeTypeIndex, 1 + badgeTypeIndex).addClass('active');
-            $.field('amount_extra').val(badgeType.extra || 0).trigger('change');
             (badgeType.onClick || $.noop)();
+            if (badgeType.extra !== null) {
+                $.field('amount_extra').val(badgeType.extra || 0).trigger('change');
+            }
```

Comment:
This was the other problem with group prereg.  Triggering the ``change`` event caused the amount to be reset back to zero, which itself caused the badge type to go back to Attendee.  Whoops!